### PR TITLE
Create vlab

### PR DESCRIPTION
**Pull request by natura4917**

### CODE REVIEW

File: domains/vlab [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+"description": "Personal Website",
```
Comment (MaskDuck) [inline domains/vlab:2]:
```suggestion
  "description": "Personal Website",
```
should be indented

**Review by MaskDuck — CHANGES_REQUESTED** [review]
also the file name should ends with `.json`

File: domains/vlab (hunk 1 above)
Comment (natura4917) [reply domains/vlab]: FixedFixed

File: domains/vlab.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Personal Website",
+  "repo": "https://github.com/natura4917/",
+  "owner": {
+    "username": "natura4917",
+    "email": "natura4917@gmail.com"
+  },
+  "record": {
+        "A": "152.67.216.65"
+    }
```
Comment (MaskDuck) [inline domains/vlab.json:10]:
```suggestion
  }
```

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Review by MaskDuck — DISMISSED** [review]
noice

**Comment by natura4917** [comment]
#1828

**Review by TheBotlyNoob — APPROVED** [review]

**Pull request merged**